<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>findwindow</class>
 <widget class="QMainWindow" name="findwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTableWidget" name="BaseTable">
      <property name="editTriggers">
       <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
      </property>
      <property name="dragDropOverwriteMode">
       <bool>true</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::DragDropMode::NoDragDrop</enum>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Дата</string>
       </property>
       <property name="background">
        <color>
         <red>240</red>
         <green>253</green>
         <blue>0</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Индекс ХТ</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Код напр.</string>
       </property>
       <property name="textAlignment">
        <set>AlignHCenter|AlignTop</set>
       </property>
       <property name="background">
        <color>
         <red>250</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Назв. дисцп.</string>
       </property>
       <property name="background">
        <color>
         <red>59</red>
         <green>0</green>
         <blue>255</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ФИО студента</string>
       </property>
       <property name="background">
        <color>
         <red>38</red>
         <green>255</green>
         <blue>0</blue>
        </color>
       </property>
      </column>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="lineEdit">
      <property name="text">
       <string>Укажите ниже дату начало поиска</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QDateEdit" name="dateEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="text">
       <string>Укажите ниже дату конца поиска</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QDateEdit" name="dateEdit_2"/>
    </item>
    <item row="5" column="0">
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="text">
       <string>Укажите название дисциплины</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLineEdit" name="lineEdit_4"/>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Поиск по периоду дат</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
